README.md

```diff
@@ -1,6 +1,6 @@
 # interview-preparation-kit
-Solutions to all the problems of [Interview Preparation Kit](https://www.hackerrank.com/interview/interview-preparation-kit) on HackerRank
-in C++, Java, Python and Javascript.
+This Repo includes all solutions to all the problems of [Interview Preparation Kit](https://www.hackerrank.com/interview/interview-preparation-kit) on HackerRank
```

Comment:
@maze-runnar I don't think these changes are required. Do you?

Reply:
Yeah no need for these changes. No need to update readme until their are big changes in project.